src/Dispatcher.php

```diff
@@ -11,6 +11,7 @@
 use Penny\Exception\MethodNotAllowed;
 use Penny\Exception\RouteNotFound;
 use Psr\Http\Message\RequestInterface;
+use Penny\Route\RouteInfo;
 
 class Dispatcher
```

Comment:
Oh yes sorry.. I confused line.. Yes we can call it Router..

> The `Router` `dispatches` a `RouterInfo`
is it a good idea? :)